.circleci/config.yml

```diff
@@ -261,10 +261,10 @@ workflows:
           filters:
             branches:
               ignore: client-java-release-branch
-      - test-behaviour-graql-reasoner:
-          filters:
-            branches:
-              ignore: client-java-release-branch
+#      - test-behaviour-graql-reasoner:
+#          filters:
+#            branches:
+#              ignore: client-java-release-branch
```

Comment:
We don't need to comment these lines out if we're already commenting them out of the workflow definition.